main.ts

```diff
@@ -1,47 +1,70 @@
 import { App, normalizePath, Notice, Plugin, PluginSettingTab, Setting, TAbstractFile, TFile, TFolder } from "obsidian";
 
 interface TemplateFile {
-  plugin: "core" | "templater";
+  plugin: "core" | "templater-obsidian";
   path: string;
 }
 
 interface HotkeysForTemplateSettings {
-  files: string[];
-  templaterFiles: string[];
+  coreFiles: string[],
+  templaterFiles: string[],
 }
 
 const DEFAULT_SETTINGS: HotkeysForTemplateSettings = {
-  files: [],
-  templaterFiles: []
+  coreFiles: [],
+  templaterFiles: [],
 };
 
 export default class HotkeysForTemplates extends Plugin {
   settings: HotkeysForTemplateSettings;
+
+  activePlugins: string[] = [];
+  templaterFolderPath: string;
+  templaterFolder: any;

```

Comment:
Why do you make them `any`? Shouldn't they be `TFolder`? And why add the extra variable `templaterFolderPath` since the path is available from the folder variable too?